de.fu_berlin.inf.dpp.core/build.gradle

```diff
@@ -0,0 +1,69 @@
+dependencies {
+  releaseDep 'commons-codec:commons-codec:1.3'
+  releaseDep 'commons-io:commons-io:2.0.1'
+  releaseDep 'commons-lang:commons-lang:2.4'
+  releaseDep 'javax.jmdns:jmdns:3.4.1'
+  releaseDep 'org.picocontainer:picocontainer:2.11.2'
+  releaseDep 'org.igniterealtime.smack:smack:3.4.1'
+  releaseDep 'org.igniterealtime.smack:smackx:3.4.1'
+  releaseDep 'xpp3:xpp3:1.1.4c'
+  releaseDep 'com.thoughtworks.xstream:xstream:1.4.10'
+  releaseDep 'org.gnu.inet:libidn:1.15'
+  releaseDep log4jVersionCore
+  releaseDep log4jVersionApi
+  // The used version is a SNAPSHOT version which is not available!
+  // TODO: use real release
```

Comment:
Nope, our jar file is a patched one.

Reply:
Ah, thank you. I already wondered why someone would use a SNAPSHOT version.
Do you know where the patched sources are stored? I did not found them in a `/patches` or `/ext-src` directory.